src/image.py

```diff
@@ -129,3 +131,49 @@ def calc_equalize_hist(self):
                 'Error: the implicit image must be on BGR color space')
         im = cv2.cvtColor(self._img, cv2.COLOR_BGR2GRAY)
         return cv2.equalizeHist(im)
+
+    def get_background(self, height_ratio=10, distance_between_peaks=10):
+        """
+        Get the background of the image if it has
+        Args:
+            - height_ratio:
+            - distance_between_peaks:
+        Returns:
+            The image background
+        """
+        def my_find_peaks(array, height_ratio, distance_between_peaks):
+            peaks, _ = find_peaks(array,
+                                  height=numpy.max(array) / height_ratio,
+                                  distance=distance_between_peaks)
+            return peaks
+
+        if self._color_space != 'BGR':
+            raise ValueError(
+                'Error: the implicit image must be on BGR color space')
+
+        gray_image = cv2.cvtColor(self._img, cv2.COLOR_BGR2GRAY)
+
+        col_var = numpy.abs(numpy.gradient(gray_image.mean(0)))
+
+        left_cd = my_find_peaks(col_var[:len(col_var) // 2], height_ratio,
+                                distance_between_peaks)
+        left = left_cd[0]
+
+        right_cd = my_find_peaks(col_var[len(col_var) // 2:], height_ratio,
+                                 distance_between_peaks) + len(col_var) // 2
+        right = right_cd[-1]
+
+        row_var = numpy.abs(numpy.gradient(gray_image.mean(1)))
+
+        top_cd = my_find_peaks(row_var[:len(row_var) // 2], height_ratio,
+                               distance_between_peaks)
+        top = top_cd[0]
+
+        bottom_cd = my_find_peaks(row_var[len(row_var) // 2:], height_ratio,
+                                  distance_between_peaks) + len(row_var) // 2
+        bottom = bottom_cd[-1]
+
+        mask = numpy.zeros(shape=(self._img.shape[0], self._img.shape[1]),
+                           dtype=numpy.uint8)
+        mask[top:bottom, left:right] = 255
+        return mask
```

Comment:
I would not couple the background remover so much with the image class. Instead, I would maintain the `src/Background_remover.py`, in which different strategies can be implemented. Then I would pass an strategy instance as an initialization parameter to the image class.

Also, I think you got the purpose of the function wrong. It does not yield the background of the image, it instead returns the mask where the ones are foreground and the zeros are the background.

My opinion is that we should maintain the "pipelineable block" fashion, thus returning the cropped image instead of the mask, as it did the method of the `src/Background_remover.py`. Also, when needed, store the mask under the path of convenience.

Reply:
You're right :) nevertheless i'll do a new PR to change some style names from your background_remover